repose-aggregator/components/services/http-client-service/http-client-service-impl/src/main/java/org/openrepose/core/services/httpclient/impl/ProxyUtilities.java

```diff
@@ -51,15 +52,17 @@ public static SSLContext getTrustingSslContext() {
     public static class AllTrustingManager implements X509TrustManager {
         @Override
         public X509Certificate[] getAcceptedIssuers() {
-            return null;
+            return new X509Certificate[0];
```

Comment:
I had the same concern. I did check the contracts, and I believe the changes I made to stop returning nulls should be legit. For the `X509TrustManager` specifically, the contract actually says this method returns:
> a non-null (possibly empty) array of acceptable CA issuer certificates.